docs/data/flink_connectors.yml

```diff
@@ -57,12 +57,19 @@ opensearch:
   source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-opensearch-1.0.0/flink-connector-opensearch-1.0.0-src.tgz.sha512"
   compatibility: ["1.16.x"]
 
-pulsar:
+pulsar3:
   name: "Apache Flink Pulsar Connector 3.0.0"
   source_release_url: "https://www.apache.org/dyn/closer.lua/flink/flink-connector-pulsar-3.0.0/flink-connector-pulsar-3.0.0-src.tgz"
   source_release_asc_url: "https://downloads.apache.org/flink/flink-connector-pulsar-3.0.0/flink-connector-pulsar-3.0.0-src.tgz.asc"
   source_release_sha512_url: "https://downloads.apache.org/flink/flink-connector-pulsar-3.0.0/flink-connector-pulsar-3.0.0-src.tgz.sha512"
   compatibility: ["1.16.x"]
+  
+pulsar4:
```

Comment:
minor: `pulsar3` and `pulsar4` are a little confused for me as it likes connector for Apache pulsar v3/v4,   we used `elasticsearch-5/6/7` to supports different Elasticsearch versions.  Maybe `pulsar-connector-v3/v4` is more clear.

Reply:
@leonardBang That's a fair comment, I've used the same setup as we've done for AWS. @dannycranmer WDYT? We can also change this later but curious on your thoughts